<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ILLEnergyTransfer</class>
 <widget class="QWidget" name="ILLEnergyTransfer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="loMain">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="MantidQt::MantidWidgets::MWRunFiles" name="rfInput" native="true">
        <property name="label" stdset="0">
         <string>Run File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCalibration">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QCheckBox" name="ckUseCalibration">
        <property name="text">
         <string>Use Calibration</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MantidQt::MantidWidgets::DataSelector" name="dsCalibration" native="true">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="autoLoad" stdset="0">
         <bool>true</bool>
        </property>
        <property name="showLoad" stdset="0">
         <bool>false</bool>
        </property>
        <property name="workspaceSuffixes" stdset="0">
         <stringlist>
          <string>_calib</string>
         </stringlist>
        </property>
        <property name="fileBrowserSuffixes" stdset="0">
         <stringlist>
          <string>_calib.nxs</string>
         </stringlist>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbGrouping">
     <property name="title">
      <string>Grouping</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="cbGroupingType">
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="swGroupingTypes">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pgILLMapDefault"/>
        <widget class="QWidget" name="pgILLMapFile">
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="MantidQt::MantidWidgets::MWRunFiles" name="rfMapFile" native="true">
            <property name="label" stdset="0">
             <string></string>
            </property>
            <property name="fileExtensions" stdset="0">
             <stringlist>
              <string>.xml</string>
             </stringlist>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOptions">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="ckMirrorMode">
        <property name="text">
         <string>Use Mirror Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MantidQt::MantidWidgets::DataSelector</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/DataSelector.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>ckUseCalibration</tabstop>
  <tabstop>cbGroupingType</tabstop>
  <tabstop>ckMirrorMode</tabstop>
  <tabstop>ckPlot</tabstop>
  <tabstop>ckSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbGroupingType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>swGroupingTypes</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>80</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>343</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ckUseCalibration</sender>
   <signal>toggled(bool)</signal>
   <receiver>dsCalibration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>87</x>
     <y>111</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
